pkg/configure/configure.go

```diff
@@ -178,6 +178,10 @@ func createUpdatedSecret(reconfigureRequest request, oldSecret corev1.Secret) co
 
 	for fullFilePathname, encodedCert := range reconfigureRequest.Certs {
 		certName := strings.Split(fullFilePathname, "/")[len(strings.Split(fullFilePathname, "/"))-1]
+		// We must strip out cluster provisioned CA certs, as they will be mounted into the pod from separate config maps
+		if certName == "service-ca.crt" || certName == "ca-bundle.crt" {
```

Comment:
Won't config-tool be a better place for this filtering? Why does the operator need to know about this strangeness of config-tool?

Reply:
If reconfigure requests from the config editor, then this problem only exists in an operator setting. However, since we are going to most likely deprecate the config-tool, we can move it there for the time being.